src/components/Table/TableColumnSelector.tsx

```diff
@@ -153,60 +140,60 @@ export const TableColumnSelector = function <T extends TableColumnState>(
 	const canAddTagColumn = !failedAddColumnRule;
 	const addTagColumnTitle = failedAddColumnRule && failedAddColumnRule.message;
 
-	return (
-		<DropDownButton
-			joined
-			alignRight
-			primary
-			plain
-			label={<SmallFontAwesomeIcon icon={faCog} />}
-		>
-			{filter(props.columns, (col) => !col.locked)
-				.map((col) => (
-					<Item
+	const memoizedItems = React.useMemo(
+		(): DropdownOption[][] => [
+			filter(columns, (col) => !col.locked).map((col) => ({
```

Comment:
I know it was already like this but, let's use the normal filtrer here, no need to use lodash